rttest/src/rttest.cpp

```diff
@@ -754,23 +760,25 @@ int Rttest::calculate_statistics(struct rttest_results * output)
     return -1;
   }
 
-  output->min_latency = *std::min_element(
-    this->sample_buffer.latency_samples.begin(), this->sample_buffer.latency_samples.end());
-  output->max_latency = *std::max_element(
-    this->sample_buffer.latency_samples.begin(), this->sample_buffer.latency_samples.end());
+  std::vector<int64_t> latency_samples(
+    this->sample_buffer.latency_samples.begin() + this->cleared_iteration + 1,
+    this->sample_buffer.latency_samples.end());
+
+  output->min_latency = *std::min_element(latency_samples.begin(), latency_samples.end());
+  output->max_latency = *std::max_element(latency_samples.begin(), latency_samples.end());
   output->mean_latency = std::accumulate(
-    this->sample_buffer.latency_samples.begin(),
-    this->sample_buffer.latency_samples.end(), 0.0) / this->sample_buffer.latency_samples.size();
+    latency_samples.begin(),
+    latency_samples.end(), 0.0) / latency_samples.size();
 
   // Calculate standard deviation and try to avoid overflow
-  output->latency_stddev = calculate_stddev(this->sample_buffer.latency_samples);
+  output->latency_stddev = calculate_stddev(latency_samples);
 
   output->minor_pagefaults = std::accumulate(
-    this->sample_buffer.minor_pagefaults.begin(),
+    this->sample_buffer.minor_pagefaults.begin() + this->cleared_iteration + 1,
```

Comment:
Why are we adding 1 here?  Won't that always skip a sample somewhere along the way?